src/main/java/com/powsybl/openloadflow/ac/outerloop/DistributedSlackOuterLoop.java

```diff
@@ -56,7 +56,8 @@ public void initialize(AcOuterLoopContext context) {
     @Override
     public OuterLoopResult check(AcOuterLoopContext context, ReportNode reportNode) {
         double slackBusActivePowerMismatch = context.getLastSolverResult().getSlackBusActivePowerMismatch();
-        boolean shouldDistributeSlack = Math.abs(slackBusActivePowerMismatch) > slackBusPMaxMismatch / PerUnit.SB;
+        double absMismatch = Math.abs(slackBusActivePowerMismatch);
+        boolean shouldDistributeSlack = absMismatch > slackBusPMaxMismatch / PerUnit.SB && absMismatch > ActivePowerDistribution.P_RESIDUE_EPS;
```

Comment:
This part could be a separate BugFix PR... let me know...

if `slackBusPMaxMismatch` is small in comparison with `ActivePowerDistribution.P_RESIDUE_EPS`, the outerloop becomes unstable because 1/ `ActivePowerDistribution` will not distribute anything and 2/ Outerloop says something has to be distributed

Reply:
Thank you for catching this! I think we can let it in this PR.